src/app.rb

```diff
@@ -0,0 +1,149 @@
+require_relative './musicalbum'
+require_relative './gaming_methods'
+require_relative './book_methods'
+require_relative './label'
+require 'json'
+
+class App
+  attr_accessor :album_list, :genre_list, :game_list, :author_list
+
+  def initialize()
+    @music_list = []
+    @genre_list = []
+    @game_list = []
+    @author_list = []
+    @book_list = []
+    @label_list = []
+  end
+
+  def music_display
+    list_music
+  end
+
+  def genre_display
+    list_genres
+  end
+
+  def music_create
+    create_music
+  end
+
+  def game_display
+    show_games
+  end
+
+  def game_create
+    create_game
+  end
+
+  def author_display
+    show_authors
+  end
```

Comment:
- Please note that I get errors when I try to list authors. Could you make sure this method is implemented? 👍🏽
![Screenshot from 2023-01-27 21-16-21](https://user-images.githubusercontent.com/44624138/215191561-d52c51db-e654-4fdd-82d7-7d8c6afc8c79.png)